### ISSUE TITLE
Provide Aarch64 in releases

### ISSUE THREAD

**Issue by StuartIanNaylor** [badaix/snapcast#1190]
please make it easier for us noobs.
sudo apt install ./snapserver_0.27.0-1_armhf.deb
```
The following packages have unmet dependencies:
 snapserver:armhf : Depends: libasound2:armhf (>= 1.0.18) but it is not installable
                    Depends: libatomic1:armhf (>= 4.8) but it is not installable
                    Depends: libavahi-client3:armhf (>= 0.6.16) but it is not installable
                    Depends: libavahi-common3:armhf (>= 0.6.16) but it is not installable
                    Depends: libc6:armhf (>= 2.28) but it is not installable
                    Depends: libexpat1:armhf (>= 2.0.1) but it is not installable
                    Depends: libflac8:armhf (>= 1.3.0) but it is not installable
                    Depends: libgcc1:armhf (>= 1:3.5) but it is not installable
                    Depends: libogg0:armhf (>= 1.0rc3) but it is not installable
                    Depends: libopus0:armhf (>= 1.1) but it is not installable
                    Depends: libsoxr0:armhf (>= 0.1.0) but it is not installable
                    Depends: libstdc++6:armhf (>= 7) but it is not installable
                    Depends: libvorbis0a:armhf (>= 1.1.2) but it is not installable
                    Depends: libvorbisenc2:armhf (>= 1.1.2) but it is not installable
E: Unable to correct problems, you have held broken packages.
```
I did compile after getting libboost but for some reason it seems to ignore `/etc/snapserver.conf` and just use defaults?

**Comment by StuartIanNaylor** [comment]
Apols Doh I had put `source = file:///home/stuart/Nums_7dot1_24_48000.wav?name=File` but forgot the section `[stream]`
Aaarch64 is likely more common now and would be great to get a deb in releases though.

**Comment by badaix** [comment]
I'm using my RPi4 based home server for the package generation for Buster, Bullseye and Bookworm. The RPi is running a 32bit OS and thus I cannot build 64 bit packages on it. 
Today my RPi5 arrived, which will replace the RPi4 and will run on a 64 bit OS, So I guess that I can soon provide 64 bit packages, and hopefully it will not take 4 hour to build them, currently the 4gig RAM are a bottleneck, forcing me to limit parallel compilation to two threads, C++ is a memory hungry beast when it comes to compiling heavily template based code, which Snpacast indirectly is - thanks to the usage of boost.

**Comment by StuartIanNaylor** [comment]
I think now with so much from fft to ML benefiting from vector instructions 32bit is now mostly a microcontroller domain.
I managed compile real easy it was just me @badaix as didn't give the section of [stream]
I did wonder about optimisation and didn't look but march=native and such do they have any effect.
PS on a tangential topic the armv8.2 mat/mul vector instructions really do make a huge diff on Pi5.

PS compiling use zram in fact use my hack as the debian one is horrid https://github.com/StuartIanNaylor/zram-swap-config but add a 2gb dphys-swap file in case zram hits uncompressable and stops any fail.
